src/drivers/sqlite/pool.js

```diff
@@ -52,6 +55,12 @@ class Pool extends EventEmitter {
       task();
     }
   }
+
+  async end() {
+    for (const connection of this.connections) {
```

Comment:
`await Promise.allSettled(this.connections.map(async (connection) => await connection.end());` could be prefered.

Reply:
done